openedx/features/discounts/utils.py

```diff
@@ -133,11 +133,11 @@ def generate_offer_html(user, course):
                 br=HTML('<br>'),
                 banner_open=HTML(
                     '<div class="first-purchase-offer-banner" role="note">'
-                    '<span class="first-purchase-offer-banner-bold">'
+                    '<span class="first-purchase-offer-banner-bold"><b>'
```

Comment:
This is just to make the MFE render this a bit better without having CSS rules for `first-purchase-offer-banner-bold` (it's still missing on a slight text color change, but that's small potatoes).

Reply:
+1 this comment for small potatoes